### ISSUE TITLE
SMB Share for Kodi Files

### ISSUE THREAD

**Issue by DerfOh** [arka9ine/TriPi-Updater#4]
Not necessarily an issue but it would be a pretty awesome addition.

**Comment by arka9ine** [comment]
Which Kodi files are you referring to? It is easily configurable, but if it's something I feel is popular enough I can throw it into an update.

**Comment by DerfOh** [comment]
The default directories for media files, like movies, music, tv, etc in Kodi. I actually managed to get this working with a USB drive by combining these two tutorials: 
- http://elinux.org/RPi_Adding_USB_Drives#Mounting_a_Partition_at_Boot
- http://www.htpcguides.com/create-samba-share-raspberry-pi/

Basically I mounted a USB drive that houses all of my files then pointed the /etc/samba/smb,conf at the mounted drive. For some reason I had to reflash the image because I was getting some strange problems with permissions. In the end I think it was just a configuration issue on my end, thanks for the quick reply though!
